test/Intl/NumberFormat.js

```diff
@@ -141,7 +143,12 @@ const tests = [
             assert.areEqual("$1.50", formatCurrency({ currencyDisplay: "symbol" }, 1.504), "Currency display: symbol");
             assert.areEqual("$1.51", formatCurrency({ currencyDisplay: "symbol" }, 1.505), "Currency display: symbol");
             // ICU has a proper "name" currency display, while WinGlob falls back to "code"
+            if (WScript.Platform.ICU_VERSION >= 62) {
```

Comment:
Can we just say `===`? Hopefully this will be fixed in future ICU updates.

Reply:
yep, that seems reasonable